utils/build/includes/extras.json

```diff
@@ -65,5 +65,6 @@
 	"src/extras/helpers/VertexNormalsHelper.js",
 	"src/extras/helpers/WireframeHelper.js",
 	"src/extras/objects/ImmediateRenderObject.js",
-	"src/extras/objects/MorphBlendMesh.js"
+	"src/extras/objects/MorphBlendMesh.js",
+	"src/extras/postprocessing/Pass.js"
```

Comment:
Not sure about this. I would, instead, add the `THREE.Pass` code at the end of `EffectComposer.js`